### DISCUSSION TITLE
Wondering how to setup basic networking in a container.

### DISCUSSION THREAD

**Discussion by wietyy** [opencontainers/runc#3881] [Q&A]
Hello! I am starting to use RunC for a research project, and I am setting up a simple container that I can use similar to a docker container. I use almost the default `config.json` from `runc spec`, but I have modified it so the rootfs points to an Alpine Linux minirootfs I got, and i changed `readonly` to false as to install packages. I would like to be able to run commands such as `apk update` or `apk add python`, though. Any help would be great! I would also like it as simple as possible, too. Thanks!

**Comment by lifubang** [comment]
Please refer to https://github.com/opencontainers/runc/issues/201

**Reply by lifubang** [reply]
The official example is [here](https://github.com/opencontainers/runtime-spec/blob/main/config.md#example-3).
